### DISCUSSION TITLE
请问切换媒体库的时候怎么继续播放当前播放的歌曲呢？

### DISCUSSION THREAD

**Discussion by WOCYANG6437** [zhongyang219/MusicPlayer2#456] [General]
首先感谢作者，这个播放器很好用，会继续用下去的。
目前有个疑惑， 例如我现在听王菲这个目录下的歌，当我切换成伍佰目录后，王菲的歌就停止了。
怎么设置成切换媒体库后，当前听的歌继续播放？
![QQ截图20221105084215](https://user-images.githubusercontent.com/117492128/200093421-15e6fb54-64ff-4e05-96c6-c48b732d6b78.png)

**Comment by lrisora** [comment]
MusicPlayer2主界面的这个列表目前兼任歌单和播放列表功能，不在这个列表内的歌曲无法播放。这一点暂且难以改变
不过如果只是想浏览歌曲的话可以使用媒体库窗口，其中提供了各种筛选功能
想手动分类的话推荐手动新建播放列表，媒体库的播放列表标签页提供播放列表的预览（文件夹模式没有预览）
